src/main/java/appeng/core/Registration.java

```diff
@@ -302,7 +229,8 @@ public void initialize( @Nonnull final FMLInitializationEvent event, @Nonnull fi
 		final IRegistryContainer registries = api.registries();
 
 		ApiDefinitions definitions = api.definitions();
-		definitions.getRegistry().getBootstrapComponents().forEach( b -> b.initialize( event.getSide() ) );
+		definitions.getRegistry().getBootstrapComponents( IInitComponent.class ).forEachRemaining( b -> b.initialize( event.getSide() ) );
+
 		//
 		// // Perform ore camouflage!
 		// ItemMaterial.instance.makeUnique();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)